persistentworkers/src/main/java/persistent/bazel/client/WorkerKey.java

```diff
@@ -144,26 +123,28 @@ public int hashCode() {
   private int calculateHashCode() {
     // Use the string representation of the protocolFormat because the hash of the same enum value
     // can vary across instances.
-    return Objects.hash(
-        cmd,
-        args,
-        env,
-        execRoot,
-        mnemonic,
-        cancellable,
-        sandboxed
-    );
+    return Objects.hash(cmd, args, env, execRoot, mnemonic, cancellable, sandboxed);
   }
 
   // Not as cool as using Bazel CommandFailureUtils
   @Override
   public String toString() {
-    return "WorkerKey(" + "\n\t"
-        + "cmd=" + cmd + ",\n\t"
-        + "args=" + args + ",\n\t"
-        + "env=" + env + ",\n\t"
-        + "mnemonic=" + mnemonic + ",\n\t"
-        + "execRoot=" + execRoot.toAbsolutePath()
+    return "WorkerKey("
+        + "\n\t"
+        + "cmd="
+        + cmd
+        + ",\n\t"
+        + "args="
+        + args
+        + ",\n\t"
+        + "env="
+        + env
+        + ",\n\t"
+        + "mnemonic="
+        + mnemonic
+        + ",\n\t"
+        + "execRoot="
+        + execRoot.toAbsolutePath()
```

Comment:
oof, some noformat here maybe?

Reply:
There's no `noformat` for google-java-format :dead:
https://github.com/google/google-java-format/wiki/FAQ#why-cant-i-use-magic-comments-to-disable-formatting-for-certain-sections-of-code

I can convert it to a Lombok `@ToString`?

Reply:
Yeah, that looks like a better idea